api/src/main/java/org/pmiops/workbench/api/CohortBuilderController.java

```diff
@@ -22,6 +23,8 @@
 import java.util.stream.Collectors;
 
 @RestController
+@UsesDefaultCdr
```

Comment:
I'm a little fuzzy on this annotation. Is the just temporary until we determine how the client will specify the cdr version(url or request params.. etc)?

Reply:
Yes, for CohortBuilderController.

I'm less sure about data browser... depends on whether we want the data browser UI to let users toggle between different CDR versions, or always use the default.

Reply:
Researchers might care about seeing how the data in CDR releases has progressed over time.

We also may have different CDR versions for registered and controlled, and the data in data browser may vary slightly for them, as discussed over Slack.

Anyway, I don't think we should worry about it at all for now... eventually, it might become a thing.